src/components/ButtonPanel.js

```diff
@@ -1,44 +1,54 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-const ButtonPanel = () => {
+
+const ButtonPanel = props => {
   const styles = {
     height: '100px',
     display: 'flex',
   };
+  const clickHandler = buttonName => {
+    props.clickHandler(buttonName);
+  };
+
   return (
     <div>
       <div style={styles}>
-        <Button color="#c9c5c5" name="AC" />
-        <Button color="#c9c5c5" name="+" />
-        <Button color="#c9c5c5" name="%" />
-        <Button name="÷" />
+        <Button color="#c9c5c5" name="AC" onClick={clickHandler} />
```

Comment:
[optional] Considering that 'onClick' is the name of react's default button event listener you might want to consider renaming for readability